drivers/rpmsg/rpmsg_gpio.c

```diff
@@ -0,0 +1,190 @@
+/*
```

Comment:
Please add a description in the commit log. Please include information on the firmware required for the driver to function and steps for testing it.